src/NuGet.Clients/NuGet.VisualStudio.Client/.vsixinclude

```diff
@@ -38,7 +38,11 @@ Microsoft.Extensions.FileSystemGlobbing.dll
 Microsoft.Extensions.Primitives.dll
 Microsoft.Web.XmlTransform.dll
 Newtonsoft.Json.dll
-System.Buffers.dll
-System.Memory.dll
 System.Numerics.Vectors.dll
-System.Runtime.CompilerServices.Unsafe.dll
+Microsoft.VisualStudio.Linux.ConnectionManager.Store.dll
+Microsoft.VisualStudio.Markdown.Core.dll
+Microsoft.VisualStudio.Markdown.Platform.dll
+System.Drawing.Common.dll
+Microsoft.Internal.VisualStudio.Shell.Framework.resources.dll
```

Comment:
Seems like you're on the right track moving some of these to the vsixignore. 
Have you tried moving `Microsoft.Internal.VisualStudio.Shell.Framework.resources.dll` as well?